Common/src/config_structure.cpp

```diff
@@ -757,8 +757,8 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   addStringListOption("MARKER_FLUID_INTERFACE", nMarker_Fluid_InterfaceBound, Marker_Fluid_InterfaceBound);
   /*!\brief MARKER_INTERFACE\n DESCRIPTION: Zone interface boundary marker(s) \ingroup Config*/
   addStringListOption("MARKER_INTERFACE", nMarker_InterfaceBound, Marker_InterfaceBound);
-  /*!\brief MARKER_FSI_INTERFACE \n DESCRIPTION: FSI interface boundary marker(s) \ingroup Config*/
-  addStringListOption("MARKER_FSI_INTERFACE", nMarker_FSIinterface, Marker_FSIinterface);
+  /*!\brief MARKER_FSI_INTERFACE \n DESCRIPTION: ZONE interface boundary marker(s) \ingroup Config*/
+  addStringListOption("MARKER_ZONE_INTERFACE", nMarker_ZoneInterface, Marker_ZoneInterface);
   /*!\brief MARKER_DIRICHLET  \n DESCRIPTION: Dirichlet boundary marker(s) \ingroup Config*/
```

Comment:
Yep, I understand, however also FSI_INTERFACE was originally designed for FSI interfaces and has been proven to be useful in other contexts...
As an example, and for the same purpose as you have define above, you could want to define boundary conditions on different zones which indeed have different physics. An example, you may want to transfer heat, instead of fluid tractions, into a structural domain. I was thinking of it as a way to generalize the way the transfer (child) classes are initialized.
Another question, I don't understand what do you mean with "owned by solver_direct_mean". When you define a config option, it is owned by the config class, that means, it should be accessible anywhere that config is available. Am I right?